src/GafferImageUI/ImageView.cpp

```diff
@@ -393,6 +393,20 @@ bool ImageView::keyPress( const GafferUI::KeyEvent &event )
 				);
 				return true;
 			}
+			if(event.key == "Home")
+			{
+				
+				V2i viewport = viewportGadget()->getViewport();
+				V2i viewportCenter(viewport.x / 2, viewport.y / 2);
+				Box3f bound = m_imageGadget->bound();
+				V2i imageCenter((bound.max.x - bound.min.x) / 2, (bound.max.y - bound.min.y) / 2);
+				Box3f frame(
+					Imath::V3f(imageCenter.x - viewportCenter.x, imageCenter.y - viewportCenter.y, 0),
```

Comment:
I really struggled to understand why this did the right thing because the distance between the two centres doesn't seem to be a meaningful quantity for this purpose. Then I realised that because the viewport happens to have the origin at `0,0`, `viewportCenter` is actually `halfViewportSize` and then everything made sense. Could you update the variable naming to make this clear?